templates/erp/includes/filters.html

```diff
@@ -0,0 +1,6 @@
+<div class="d-flex row p-2">
+    {% include "search/tag_with_label.html" with slug="visual" input_name="erp_filter" eventType="filterClicked" name="Difficulté à voir" icon="diff-vision" %}
+    {% include "search/tag_with_label.html" with slug="walking" input_name="erp_filter" eventType="filterClicked" name="Difficulté à marcher" icon="diff-walking" %}
+    {% include "search/tag_with_label.html" with slug="hearing" input_name="erp_filter" eventType="filterClicked" name="Difficulté à entendre" icon="deaf" %}
+    {% include "search/tag_with_label.html" with slug="understand" input_name="erp_filter" eventType="filterClicked" name="Difficulté à comprendre" icon="diff-understand" %}
+</div>
```

Comment:
Are the names translated later on? Looks like no
We might have to create vars at this point.